tslearn/bases.py

```diff
@@ -0,0 +1,217 @@
+from abc import ABCMeta, abstractmethod
+from sklearn.base import BaseEstimator
+from . import hdftools
+import json
+import pickle
+import numpy as np
+
+
+class BaseModelPackage(BaseEstimator, metaclass=ABCMeta):
+    @abstractmethod
+    def is_fitted(self) -> bool:
```

Comment:
Type annotations would break Python2 compatibility. Though I know Python2 is not very important, I would prefer we keep compatibility when possible.

Reply:
Also, as the method will be implemented by all models to be serialized, it would be nice to make those function protected members of the class so that they do not appear in their docs (they are useless for the end user).

Would it be possible to rename it to `_is_fitted`?